### ISSUE TITLE
unpublish not working?

### ISSUE THREAD

**Issue by micuat** [2bbb/ofxPubSubOsc#34]
I'm not sure what I'm doing but

```
ofxUnpublishOsc(lastIp, lastPort, "/n/osc/fps");
ofxUnpublishOsc(lastIp, lastPort, "/n/osc/rot");
ofxPublishOsc(ip, port, "/n/osc/fps", &ofGetFrameRate);
ofxPublishOsc(ip, port, "/n/osc/rot", quat);
```

and I'm still receiving messages on `lastIp:lastPort`. It's on iOS. I will check on win/osx if I can replicate the problem.

**Comment by 2bbb** [comment]
omg...

https://github.com/2bbb/ofxPubSubOsc/blob/master/src/ofxOscPublisher.h#L804-L806

**Comment by 2bbb** [comment]
@micuat 

i fixed this on `master` and `v0.3.0beta`.
thanks for report!!

**Comment by micuat** [comment]
thanks it's working!
